Azure/azure-cli
# [App Service] `az functionapp create`: Add DAPR support for Centauri apps

**Related command**
`az functionapp create`
`az functionapp config container set`

**Description**
We want to support DAPR for Azure Functions on container app environments. 

**Testing Guide**
1. Create the container app environment `az containerapp env create`
2. Create function app in container app environment and configure DAPR 'az functionapp create -g {} -n {} -s {} --environment {} --dapr-app-id daprappid --dapr-app-port 800 --dhmrs 4 --dhrbs 50 --dapr-log-level debug --enable-dapr true --functions-version 4`
3. Update DAPR configuration for existing function app in container app environment `az functionapp config container set -g {} -n {} --dapr-app-id daprappid1 --dapr-app-port 80 --dal --dhmrs 6 --dhrbs 60 --dapr-log-level warn --enabled-dapr false`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
